### ISSUE TITLE
Flakiness in waiter

### ISSUE THREAD

**Issue by netoneko** [orbs-network/orbs-network-go#292]
```
--- FAIL: TestPublicApiWaiter_WaitFor (0.02s)
	require.go:1185: 
			Error Trace:	waiter_test.go:100
			            				context.go:8
			            				waiter_test.go:93
			Error:      	Max difference between 2018-10-02 16:00:04.36063169 +0000 UTC m=+0.019245691 and 2018-10-02 16:00:04.34480587 +0000 UTC m=+0.003419865 allowed is 13ms, but difference was 15.825826ms
			Test:       	TestPublicApiWaiter_WaitFor
			Messages:   	expected not to reach timeout
```

**Comment by noambergIL** [comment]
#293

**Comment by electricmonk** [comment]
Fixes #293

Shai
________________________________
